src/resources/messages_pt_BR.properties

```diff
@@ -0,0 +1,9 @@
+LOADING_PROCESS=Carregando {0}
+RUNNING_PROCESS=Executando {0}
+ENDING_PROCESS={0} terminado. X={1}. Y={2}
+INTERRUPTING_PROCESS_1=Interrompendo {0} apos {1} instrucao
+INTERRUPTING_PROCESS_2=Interrompendo {0} apos {1} instrucoes
```

Comment:
We need to talk to our teacher about this message, because in pdf file the example has words with accent